src/vivarium_gates_nutrition_optimization_child/data/utilities.py

```diff
@@ -390,7 +395,7 @@ def filter_relative_risk_to_cause_restrictions(data: pd.DataFrame) -> pd.DataFra
     affected_measures = set(data.affected_measure)
     for cause, measure in product(affected_entities, affected_measures):
         df = data[(data.affected_entity == cause) & (data.affected_measure == measure)]
-        cause = get_entity(EntityKey(f"cause.{cause}.{measure}"))
+        cause = get_gbd_2021_entity(EntityKey(f"cause.{cause}.{measure}"))
```

Comment:
AFAICT, this is only used for the RRs that we load in with GBD 2021 age bins, so it seemed safe to change.